data_structures/linked_list.cpp

```diff
@@ -66,11 +52,14 @@ bool isDigit(const std::string &s) {
 class link
 {
 	public:
-		explicit link(int v=0, link* s=nullptr)
-		:val(v),succ(s)
+		explicit link(int v=0, link* s=nullptr) // v and s are used to initialize members value and succ respectively
+		:value(v),succ(s)
 		{}
-		int val;
-		link* succ;
+		int val(){ return value;}
+		
+		link* succ; /// pointer to the next value on the list
```

Comment:
the correct documentation protocol was explicitly provided to you in the previous reviews. please follow the standards , thank you